### PR TITLE
DOCS: SE Diffs 3.x to Main *.adoc 

### PR DESCRIPTION
**Pull request by ljamen**
### Description
This PR will capture all of the diffs from 3.x to main in se/docs. 
Each commit will reference a different adoc.

Once merged I will update the 4x doc issues indicating that the docs have been updated for diffs.

### CODE REVIEW

**Comment by ljamen** [comment]
Please note that in order to get a clean build I had to temporarily comment out 2 include files. I'm assuming they will be merged soon. 

webserver.adoc  include file not found: C:/Users/LJAMEN/Downloads/Documents/GitHub/Helidon/helidon/docs/config/io_helidon_webserver_WebServerTls.adoc

webclient.adoc include file not found: C:/Users/LJAMEN/Downloads/Documents/GitHub/Helidon/helidon/docs/config/io_helidon_webclient_WebClientConfiguration.adoc

File: docs/se/webserver.adoc [hunk 1]
```diff
@@ -328,8 +326,8 @@ To set up requested URI discovery on the default socket for your server, use the
 .Requested URI set-up for the default server socket
 ----
 import io.helidon.common.configurable.AllowList;
-import static io.helidon.http.RequestedUriDiscoveryContext.RequestedUriDiscoveryType.FORWARDED;
-import static io.helidon.http.RequestedUriDiscoveryContext.RequestedUriDiscoveryType.X_FORWARDED;
+import static io.helidon.webserver.SocketConfiguration.RequestedUriDiscoveryType.FORWARDED;
```
Comment (barchetta) [inline docs/se/webserver.adoc:329]: Revert this edit.

File: docs/se/webserver.adoc [hunk 2]
```diff
@@ -328,8 +326,8 @@ To set up requested URI discovery on the default socket for your server, use the
...
+import static io.helidon.webserver.SocketConfiguration.RequestedUriDiscoveryType.X_FORWARDED;
```
Comment (barchetta) [inline docs/se/webserver.adoc:330]: Revert this edit.

File: docs/se/webserver.adoc [hunk 3]
```diff
@@ -376,7 +374,7 @@ Your code obtains the requested URI information from the Helidon server request
 [source,java]
 .Retrieving Requested URI Information
 ----
-import io.helidon.http.UriInfo;
+import io.helidon.common.http.UriInfo;
```
Comment (barchetta) [inline docs/se/webserver.adoc:377]: Revert this edit.

File: docs/se/webserver.adoc [hunk 4]
```diff
@@ -387,7 +385,7 @@ public class MyHandler implements Handler {
     }
 }
 ----
-See the link:{http-javadoc-base-url}/io/helidon/common/http/UriInfo.html[`UriInfo`] JavaDoc for more information.
+See the link:{common-http-javadoc-base-url}/io/helidon/common/http/UriInfo.html[`UriInfo`] JavaDoc for more information.
```
Comment (barchetta) [inline docs/se/webserver.adoc:388]: Revert this edit.

File: docs/se/webserver.adoc [hunk 5]
```diff
@@ -962,34 +960,8 @@ WebServer.builder()
          .build();
 ----
 
-== TLS configuration
```
Comment (barchetta) [inline docs/se/webserver.adoc:965]: Revert this edit (put this section back)

**Review by barchetta — APPROVED** [review]

**Pull request merged**